README.md

```diff
@@ -43,6 +43,12 @@ You can use the demo wallet to test Regulated Assets ([SEP-8]), Hosted Deposit a
 9. Leave the pop-up window open while you wait to see the deposit of SRT go through - you can close when you see “Status” is complete and you have SRT.
 
 ### Demo-ing Cross-Border Payments ([SEP-31]) on Testnet
+
+_Note: specifically in the case of demo-ing SEP-31 in the Demo Wallet, notice
+the public and secret keys don't represent the Sending Client but instead the
+Sending Anchor's account. In SEP-31, the only Stellar transaction happening is
+between the Sending and the Receiving anchors._
+
```

Comment:
Is there anything we can do to make this clearer in the UI too? I know personally that I'm likely to miss a note like this and make the wrong assumptions in the UI.

Reply:
Good idea, I can add a tooltip to the UI!

Reply:
Done in https://github.com/stellar/stellar-demo-wallet/pull/179/commits/7846ed058ccd19391c73060ac38518e82c3c3b8b and updated the PR description with a screenshot.